test-apps/test-watchdog/src/main.cpp

```diff
@@ -1,12 +1,21 @@
 #include "WatchdogWrapper.h"
 #include "mbed.h"
 
+Thread pet_thread;
 std::chrono::milliseconds countdown_ms = 1000ms;
 std::chrono::milliseconds pet_ms       = 200ms;
 
+void periodic(std::chrono::milliseconds *pet_ms) {
```

Comment:
```suggestion
void pet_dog_task(std::chrono::milliseconds *pet_ms) {
```
Let's give this a descriptive name